WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -268,6 +268,8 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
             editorContentWasUpdated()
```

Comment:
looks like we don't need this line anymore

Reply:
Right. Thank you!